JASP-Engine/JASP/R/robustbayesianmetaanalysis.R

```diff
@@ -833,22 +834,24 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
   
   
   ### create overview table
-  overall_summary <-
-    createJaspTable(title = gettext("Model Summary"))
+  overall_summary <- createJaspTable(title = gettext("Model Summary"))
   overall_summary$position <- 1
   
   overall_summary$addColumnInfo(name = "terms",     title = "",                type = "string")
   overall_summary$addColumnInfo(name = "models",    title = gettext("Models"), type = "string")
   overall_summary$addColumnInfo(name = "priorProb", title = gettext("P(M)"),   type = "number")
-  
-  for (i in 1:nrow(fitSummary[["overview"]])) {
-    temp_row <- list(
-      terms     = if (i == 1) gettext("Effect") else if (i == 2) gettext("Heterogeneity") else if (i == 3) gettext("Publication bias"),
-      models    = paste0(fitSummary[["overview"]][["Models"]][i], "/", fitSummary[["add_info"]][["n_models"]]),
-      priorProb = fitSummary[["overview"]][["Prior prob."]][i]
-    )
-    
-    overall_summary$addRows(temp_row)
+
+  if (!options[["measures"]] == "fitted") {
+    for (i in 1:nrow(fitSummary[["overview"]])) {
```

Comment:
If this is always `1:3`, you could also write that. If not please ignore this comment.

Reply:
for now, but it not might be in the future - I'm planning on adding another possible factor crossed with these, so I would rather keep it without plugging in numbers